st2api/st2api/listener.py

```diff
@@ -49,6 +44,11 @@ def __init__(self, connection):
 
     def get_consumers(self, consumer, channel):
         return [
+            consumer(queues=[announcement.get_queue(routing_key=publishers.ANY_RK,
```

Comment:
an exclusive Q without a unique name is asking for trouble. Anyway this is not a problem for your change specifically so I will take care of it later.